com.unity.render-pipelines.universal/Runtime/IntermediateTextureMode.cs

```diff
@@ -10,7 +10,7 @@ public enum IntermediateTextureMode
         /// </summary>
         Auto,
         /// <summary>
-        /// Forces rendering via an intermediate texture, enabling compatibility with renderer features that do not declare their needed inputs, but can have a significant performance impact on some platforms.
+        /// Forces rendering via an intermediate texture if any Render Feature is active, enabling compatibility with renderer features that do not declare their needed inputs, but can have a significant performance impact on some platforms.
```

Comment:
```suggestion
        /// Forces rendering via an intermediate texture if any Renderer Feature is active, enabling compatibility with renderer features that do not declare their needed inputs, but can have a significant performance impact on some platforms.
```

Reply:
Fixed in [9ca0cfb](https://github.com/Unity-Technologies/Graphics/pull/7280/commits/9ca0cfb17990d6f149f2b0f1ba5f29b4b1cfd884)